### DISCUSSION TITLE
List of New Features in Netbeans IDE

### DISCUSSION THREAD

**Discussion by lyh061619** [apache/netbeans#8092] [Ideas]
### Description

Through my daily work, I often use the Eclipse IDE and IDEA to achieve a satisfactory interactive experience. The effect is excellent, and I believe that the Netbeans IDE is also an excellent integrated development tool. These experiences are indispensable.the following list of new features:

1.Click the mouse to open the file
1.1 Select on Hover
1.2 Open when using arrow keys
![e2](https://github.com/apache/netbeans/assets/15118571/b9987df7-821c-43cd-9f75-2d895bddbc6c)

2.Show debug values inline on text editors

3.Display the number of times classes and methods have been referenced. Click to navigate to the specific location of the reference
![s1](https://github.com/apache/netbeans/assets/15118571/15de733d-59f3-44f0-abc5-749fecb073cb)

4.Always show full workspace path in windows header title

5. Also provides SVN/Git modification info on the class/methods

### Use case/motivation

Better interactive experience, improving work efficiency

### Are you willing to submit a pull request?

No

**Comment by hantsy** [comment]
I think this can be found by `Find Usages`.

VScode, IDEA also provides Git modification info on the class/methods.

**Comment by Chris2011** [comment]
Due to this is a generic list I converted this to a discussion. Yes, all suggestions are very nice and in one or two I already looked into it, but not on my prio list atm due to other stuff. And also all of the things are independent, so please create a ticket for each feature and not a collectors ticket. Thx.
